src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
+        int amountIndex = 1;
+        int arraysMaxSize = 3;
+        int resultIndex = 2;
+        String[] type = new String[arraysMaxSize];
+        int[] amount = new int[arraysMaxSize];
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String value = reader.readLine();
+            while (value != null) {
```

Comment:
The use of `arraysMaxSize` as a limit for the `type` and `amount` arrays might not be flexible enough to handle a variable number of operation types. Consider using a `Map<String, Integer>` to dynamically handle different operation types without a fixed array size.